src/application/service/auth.service.ts

```diff
@@ -24,15 +19,17 @@ export class AuthService implements AuthUseCase {
     private readonly userDbCommandPort: UserDbCommandPort,
   ) {}
 
-  async login(command: LoginCommand): Promise<IToken> {
-    const userCredential = await this.userDbQueryPort.getUserForLogin(command.email);
+  async loginWithSocial(command: SocialLoginCommand): Promise<IToken> {
+    const user = await this.userDbQueryPort.getUserBySnsId(command.snsId);
 
-    const isValid = await comparePassword(command.password, userCredential.password!);
-    if (!isValid) {
-      throw new InvalidPasswordException();
+    if (user) {
+      return this.generateTokens(user.id!);
     }
 
-    return this.generateTokens(userCredential.id);
+    const createUser = new CreateUser(command.snsId, command.nickname, command.profileImageUrl);
```

Comment:
Create User 대신 command사용해도 괜찮을 것 같아요

```suggestion
    const commend: CreateUserCommend = {
        snsId: command.snsId, 
        nickname: command.nickname, 
        profileImageUrl: command.profileImageUrl
    };
```

Reply:
command사용한 걸로 수정했습니다!